### PR TITLE
Proofread string grammar

### PR DESCRIPTION
**Pull request by dgreenhalgh**
Cleaned up string typos, replaced hardcoded strings, and subsequently added to our eToast code

### CODE REVIEW

**Comment by reidbaker** [comment]
Looks perfect! Next person pull this in :octocat:

File: SoundStream/src/com/lastcrusade/soundstream/util/Toaster.java [hunk 1]
```diff
@@ -34,5 +34,14 @@ public static void eToast(Context context, String s) {
         Log.e(Thread.currentThread().getStackTrace()[methodDepth].toString(), s);
         toastWorker(context, s);
     }
+    
+    public static void eToast(Context context, int resId) {
+    	Log.e(Thread.currentThread().getStackTrace()[methodDepth].toString(), context.getString(resId));
```
Comment (theJenix) [inline SoundStream/src/com/lastcrusade/soundstream/util/Toaster.java]: This should call the eToast method that takes a string, and pass in context.getString(resId) as that string.  See iToast.

**Comment by theJenix** [comment]
One small change required in Toaster.java and then it's good to :goat:

**Comment by dgreenhalgh** [comment]
Good catch, this should be good to go

**Comment by theJenix** [comment]
:shoe: in

**Pull request merged**